src/com/example/pathom.cljs

```diff
@@ -66,7 +90,7 @@
 
 (def my-resolvers-and-mutations
   "Add any resolvers you make to this list (and reload to re-create the parser)"
-  [index-explorer create-random-thing i-fail person current-user user-authenticated?])
+  [index-explorer create-random-thing i-fail person current-user user-authenticated? quiz-by-id readiness-quiz])
```

Comment:
have to add our new resolvers to our pathom parser for pathom to know about the attributes it resolves.